Toolchain/default.nix

```diff
@@ -35,6 +35,7 @@ mkShell.override { stdenv = gccStdenv; } {
       imagemagick
       libtool
       meson
+      wayland-scanner
```

Comment:
For https://github.com/SerenityOS/serenity/blob/master/Ports/wayland/patches/0002-Use-native-wayland-scanner.patch